### DISCUSSION TITLE
First launch attempt with grblHal 2.0.36

### DISCUSSION THREAD

**Discussion by pierrerdurand** [phil-barrett/grblHAL-teensy-4.x#51] [Q&A]
Have a new grblHal breakout board v2.07 fully built.
Have uploaded the 'grblHAL_T41 3X qei.hex' with the Teensy loader.
Installed ioSender 2.0.36 on windows 11
Have one DM556 stepper controller connected to the X axis connections with a bi-polar CUI stepper attached to the stepper controller.
Nothing else is connected to the board other than the USB power from the windows PC.
When I launch ioSender 2.0.36 (with the DM556 attached or not attached) I get the ALARM:10 error: "E-Stop active! - clear before continuing..."
After acknowledging the message, I get E-Stop still active, exit?" message and am unable to to any other actions.
What I am doing wrong?

![20220223_144656](https://user-images.githubusercontent.com/7866347/155414421-390559fe-6a4e-4cf9-bb8e-195bfd72294c.jpg)

![20220223_144735](https://user-images.githubusercontent.com/7866347/155414612-58126974-67db-4290-999a-72ff63f99f37.jpg)

**Comment by phil-barrett** [comment]
Bon Jour, Pierre!

You are doing nothing wrong.  You are seeing a common problem.  You need to short halt to ground and then invert the halt input. Read about it [here in the wiki](https://github.com/grblHAL/core/wiki/First-Run-Grbl-Settings) and [here in my blog.](https://www.grbl.org/single-post/naked-teensy-party)

**Comment by pierrerdurand** [comment]
Thanks Phil,  they are Normally Closed with grblHal…..
I will do that.
Thanks Phil,

Eager to send photos of the CNC’s I have built.


Pierre


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
